src/main/java/com/znsio/e2e/tools/Visual.java

```diff
@@ -115,10 +116,11 @@ private com.applitools.eyes.appium.Eyes instantiateAppiumEyes(String driverType,
         try {
             appEyes.open(innerDriver, appName + "-" + platform, testName);
             LOGGER.info("instantiateAppiumEyes: Is Applitools Visual Testing enabled? - " + !appEyes.getIsDisabled());
-        } catch(IllegalArgumentException e) {
-            throw new InvalidTestDataException(String.format("Exception in instantiating Applitools for Apps: '%s;", e.getMessage(), e));
+        } catch(IllegalArgumentException | EyesException e) {
+            LOGGER.error("Issue found while instantiating Applitools eye, closing Appium driver instance");
```

Comment:
Create a common string variable for the error message. Use that string variable to log and throw the exception